tests/commands/autoupdate_test.py

```diff
@@ -114,8 +114,11 @@ def test_autoupdate_out_of_date_repo(
     )
     write_config('.', config)
 
+    runner = Runner('.', C.CONFIG_FILE)
     before = open(C.CONFIG_FILE).read()
-    ret = autoupdate(Runner('.', C.CONFIG_FILE), tags_only=False)
+    repo_name = 'file://{}'.format(out_of_date_repo.path)
+    # It will update the repo, because the name matches
+    ret = autoupdate(runner, tags_only=False, repo=repo_name)
```

Comment:
let's leave this test with the default `repo=None` and just add new tests

Reply:
👍